vendor/golang.org/x/tools/go/gcexportdata/example_test.go

```diff
@@ -0,0 +1,121 @@
+// Copyright 2016 The Go Authors. All rights reserved.
```

Comment:
Let's remove the `_test.go` files and the `testdata` directory. No need to keep these.

Reply:
Done.